molecule/default/tasks/exec.yml

```diff
@@ -0,0 +1,50 @@
+---
+- vars:
+    ns: k8s-exec
+    pod: sleep-pod
+    pod_def:
```

Comment:
Elsewhere in tests, we spell out the full name, so maybe use `exec_namespace` (above) and `exec_pod_definition` for this variable?

Reply:
Done